Sources/Common/DataModel/BoundingBox/index.js

```diff
@@ -119,7 +119,7 @@ export function setMaxPoint(bounds, x, y, z) {
   return xMax !== x || yMax !== y || zMax !== z;
 }
 
-export function inflate(bounds, delta) {
+function adjust(bounds, delta) {
```

Comment:
I'm sorry, I have not been clear. I made different suggestions regarding the names.
Here is what I had in mind:
```
function inflate(bounds, delta) {
  if (delta === undefined) {
    return nonZeroVolume(bounds);
  }
  bounds[0] -= delta;
  bounds[1] += delta;
  bounds[2] -= delta;
  ...
  return bounds
}

function minInflate||adjust||non_zero_volume(bounds) {
 ...
    return inflate(bounds, 0.5);
 ...
 return bounds;
}
```

Reply:
Excuse me(maybe it was a long day for me) but what portion of the code you put inside `adjust` if it's not :

```
bounds[0] -= delta;
...
bounds[5] += delta;
```

Reply:
if could explain more since i am little bit lost on this one also `return minInflate||adjust||nonZeroVolume(bounds);`

Reply:
Thank you